includes/admin/customers/customer-actions.php

```diff
@@ -50,12 +50,14 @@ function edd_edit_customer( $args ) {
 	$customer_info = wp_parse_args( $customer_info, $defaults );
 
 	// Sanitize the inputs
+	$address = array();
 	$address['line1']         = $customer_info['line1'];
 	$address['line2']         = $customer_info['line2'];
 	$address['city']          = $customer_info['city'];
 	$address['zip']           = $customer_info['zip'];
 	$address['state']         = $customer_info['state'];
 	$address['country']       = $customer_info['country'];
+	$customer_data = array();
```

Comment:
If we're adding these, the whitespace needs to be corrected...please adhere to WP Core coding standards as specified here:
https://make.wordpress.org/core/handbook/coding-standards/php/#indentation

Please do this for all changes in this PR. I won't make more comments on it.